authui/package.json

```diff
@@ -16,7 +16,6 @@
     "stylelint": "stylelint --cache './src/**/*.{css,scss}'",
     "dev": "vite",
     "build": "vite build --outDir '../resources/authgear/generated'",
-    "watch": "vite build --watch --outDir '../resources/authgear/generated'",
```

Comment:
But simply running `vite` will not output to `'../resources/authgear/generated'`, isn't it?

Reply:
Yes it won't. The current `dev` commands works by generating a `__generated_asset.html` which contains urls pointing to the vite development server.
For development purpose `dev` should be enough therefore we don't need `watch`. The reason I remove `watch` is it always remove the `.gitkeep` inside `resources/authgear/generated` and it is easy to commit this change accidentally.